app/controllers/formbuilder/edit.js

```diff
@@ -36,6 +37,33 @@ export default class FormbuilderEditController extends Controller {
 
   @tracked isInitialDataLoaded = false;
 
+  @tracked isShowBuilder = true;
+  @tracked fieldsInForm = [];
+  formValidationService = null;
```

Comment:
Also try to use the derived state here as [mentioned](https://github.com/lblod/frontend-form-builder/pull/22#discussion_r1344071910)

Reply:
This doesn't seem to be used (yet)? Does this block the merging of this PR?

Reply:
Definitly not :D